src/ripple/rpc/handlers/LedgerEntry.cpp

```diff
@@ -598,11 +598,11 @@ doLedgerEntry(RPC::JsonContext& context)
             else
                 uNodeIndex = keylet::did(*account).key;
         }
-        else if (context.params.isMember(jss::mpt_issuance_id))
+        else if (context.params.isMember(jss::mpt_issuance))
```

Comment:
Isn't this still issuance_id?

Reply:
It still is. I renamed it so that the field name is the object type instead because all the other [account_root](https://xrpl.org/ledger_entry.html#get-accountroot-object)
[amm](https://xrpl.org/ledger_entry.html#get-amm-object) 
[directory](https://xrpl.org/ledger_entry.html#get-directorynode-object) are object types like account_root, amm, directory, etc